libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -602,6 +602,24 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
                     add_motor(AP_MOTORS_MOT_6,-150, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  4);
                     success = true;
                     break;
+                case MOTOR_FRAME_TYPE_FTC_PLUS:
```

Comment:
This is incorrect for + configuration.  You've described X configuration twice.

Reply:
I don't really think this is appropriate for a + configuration as it only really has symmetry as a x configuration. So I would simply not support the + config.